Dockerfile

```diff
@@ -1,27 +1,25 @@
-FROM lukemathwalker/cargo-chef:latest-rust-1.72.0-buster AS chef
+FROM rust:alpine as chef
+RUN apk add --no-cache musl-dev file make
+RUN cargo install cargo-chef 
 WORKDIR app
 
 FROM chef AS planner
 COPY . .
 RUN cargo chef prepare --recipe-path recipe.json
 
 FROM chef AS builder
-
-# Ensure working C compile setup (not installed by default in arm64 images)
-RUN apt update && apt install build-essential -y
-
 COPY --from=planner /app/recipe.json recipe.json
 RUN cargo chef cook --release --recipe-path recipe.json
-
 COPY . .
 RUN cargo build --release --bin atuin
 
-FROM debian:bullseye-20230919-slim AS runtime
-
-RUN useradd -c 'atuin user' atuin && mkdir /config && chown atuin:atuin /config
-# Install ca-certificates for webhooks to work
-RUN apt update && apt install ca-certificates -y && rm -rf /var/lib/apt/lists/*
-WORKDIR app
+FROM alpine:3.12 as runner
```

Comment:
Using 3.18 would be a good up-to-date version with plenty of space left in the lifecycle.